index.d.ts

```diff
@@ -0,0 +1,50 @@
+
+    export interface SegmentedControlTtabProperties extends React.Props<ReactNativeSegmentedControlTab> {
+        // Default ['One', 'Two', 'Three'] ; titles of tabs Default 
+        values?: Array<string>
+        // Default [0] ;index of tab item to be selected initially 
+        selectedIndex?: Array<number>
+        // Default  [0];Array of indices of tab items to be selected initially - when multiple prop is true else it will take selectedIndex
+        selectedIndices?: Array<number>
+        // Default  true; Boolean to enable or disable the component
+        enable?: boolean
+        // Default  false; Boolean which enables the multiple selection option
+        multiple?: boolean
+        // Default  5;borderRadius of whole tab
+        borderRadius?: number
+        // Default  base styles added in SegmentedControlTab.js;external styles can be passed to override the default styles of the segmentedControl wrapper
+        tabsContainerStyle?: any
```

Comment:
`StyleProp<ViewStyle>` instead of any?

Reply:
This is a good idea, I am starting to make a ts d.ts, because this file I can already use, I hope to be better